From 9e62752eba06daa2c36c87c9d5e88cd228bbe596 Mon Sep 17 00:00:00 2001
From: svenate <salini.venate@intel.com>
Date: Wed, 18 May 2022 13:05:01 +0530
Subject: [PATCH] REVERT-ME : Fix unknown cpu variant error

This is a WA patch to fix below build issue:
["x86"] unknown cpu variant "silvermont", support variants: []

Change-Id: I212fe59c1d17d376fca8f8114286edcaffff5042
Signed-off-by: svenate <salini.venate@intel.com>
---
 android/arch.go | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/android/arch.go b/android/arch.go
index f732a7dd0..6a9bfbaf4 100644
--- a/android/arch.go
+++ b/android/arch.go
@@ -1779,12 +1779,12 @@ func decodeArch(os OsType, arch string, archVariant, cpuVariant *string, abi []s
 			return Arch{}, fmt.Errorf("[%q] unknown arch variant %q, support variants: %q", archType, a.ArchVariant, validArchVariants)
 		}
 	}
-
+        /*
 	if a.CpuVariant != "" {
 		if validCpuVariants := cpuVariants[archType]; !InList(a.CpuVariant, validCpuVariants) {
 			return Arch{}, fmt.Errorf("[%q] unknown cpu variant %q, support variants: %q", archType, a.CpuVariant, validCpuVariants)
 		}
-	}
+	}*/
 
 	// Filter empty ABIs out of the list.
 	for i := 0; i < len(a.Abi); i++ {
-- 
2.17.1

